### DISCUSSION TITLE
Is there support for file upload in dynamic schema?

### DISCUSSION THREAD

**Discussion by sujeetsr** [async-graphql/async-graphql#1384] [Q&A] [answered]
There is an example for regular file upload but not with dynamic schema. Is it supported?

**Comment by sunli829** [comment, marked as answer]
added in `v6.0.9`, [example](https://github.com/async-graphql/examples/tree/ed1e7f28e472b27d883092f660c7632942c4f314/poem/dynamic-upload)
